kratos/python_scripts/process_factory.py

```diff
@@ -18,31 +19,52 @@ def ConstructListOfProcesses( self, process_list ):
                 raise NameError('"python_module" must be defined in your parameters. Check all your processes')
 
             # python-script that contains the process
-            python_module_name = item["python_module"].GetString()
+            python_module_name = self._GetFullModuleNameName(item)
+            
+            python_module = import_module(python_module_name)
+            p = python_module.Factory(item, self.Model)
+            constructed_processes.append( p )
 
-            if item.Has("kratos_module"): # for Kratos-processes
-                """Location of the process in Kratos; e.g.:
-                - KratosMultiphysics
-                - KratosMultiphysics.FluidDynamicsApplication
-                """
+        return constructed_processes
+
+    def GetMapOfRequiredHistoricalVariables( self, process_list):
+        required_variables_map = defaultdict(set)
+        for i in range(0,process_list.size()):
+            item = process_list[i]
+            if not item.Has("python_module"):
+                KM.Logger.PrintWarning("Your list of processes: ", process_list)
+                raise NameError('"python_module" must be defined in your parameters. Check all your processes')
```

Comment:
can you add `item` to the print? Then it is easier to debug

Reply:
Sure, I copied it from `ConstructListOfProcesses` but I agree it makes more sense to just print the process that is missing this.